M2/Macaulay2/m2/exports.m2

```diff
@@ -1253,6 +1256,7 @@ exportMutable {
 	"printingTrailLimit",
 	"programPaths",
 	"recursionLimit",
+	"regexFlags",
```

Comment:
This variable should be eliminated: having a global variable that changes default basic behavior is a sure way to cause bugs.

Reply:
Again, this is related to the regex PR, not here. And no, it doesn't cause bugs. You can change the default to PosixPerl and M2 still loads and packages work properly.

Reply:
It will definitely cause a bug eventually.

Reply:
I removed this.